src/local/butler/scripts/migration/monorail_to_issuetracker_migrator.py

```diff
@@ -63,8 +64,8 @@ def get_monorail_issuetracker_issue_id_dictionary(file_loc):
     fieldnames = ['monorail_id', 'issuetracker_id']
     reader = csv.DictReader(csvfile, fieldnames=fieldnames)
     for row in reader:
-      monorail_issue_id = row[fieldnames[0]]
-      issuetracker_issue_id = row[fieldnames[1]]
-      issue_id_dictionary[monorail_issue_id] = issuetracker_issue_id
+      key_id = row[fieldnames[1]] if roll_back else row[fieldnames[0]]
+      value_id = row[fieldnames[0]] if roll_back else row[fieldnames[1]]
+      issue_id_dictionary[key_id] = value_id
```

Comment:
This works, but I personally find it needlessly obscure. Also, you're defining `reader` twice, shadowing the first `csv.reader` with a `csv.DictReader` who confuses things more than it helps, IMO. How about:

```suggestion
for monorail_id, issuetracker_id in reader:
  if roll_back:
    issue_id_dictionary[issuetracker_id] = monorail_id
  else:
    issue_id_dictionary[monorail_id] = issuetracker_id
```

Or:

```suggestion
issue_id_dictionary = {
  monorail_id: issuetracker_id
  for monorail_id, issuetracker_id in reader
}
  
if roll_back:
  # Reverse ID mapping.
  issue_id_dictionary = {v: k for k,v in issue_id_dictionary}
```

Reply:
Used the 1st suggestion. Thanks!